examples/asr/experimental/structured/speech_to_text_structured_v2.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import asdict
+
+import pytorch_lightning as pl
+
+from nemo.collections.asr.models import EncDecCTCModel, configs
+from nemo.core.config import modelPT, optimizers, schedulers
+from nemo.utils.exp_manager import exp_manager
+
+"""
+python speech_to_text_structured_v2.py
+"""
+
+# fmt: off
+LABELS = [
+    " ", "a", "b", "c", "d", "e",
+    "f", "g", "h", "i", "j", "k",
+    "l", "m", "n", "o", "p", "q",
+    "r", "s", "t", "u", "v", "w",
+    "x", "y", "z", "'",
+]
+
+optim_cfg = optimizers.NovogradParams(
+    lr=0.01,
+    betas=(0.8, 0.5),
+    weight_decay=0.001
+)
+
+sched_cfg = schedulers.CosineAnnealingParams(
+    warmup_steps=None,
+    warmup_ratio=None,
+    min_lr=0.0,
+)
+# fmt: on
+
+
+def main():
+    # NeMo Model config
+    cfg = modelPT.NemoConfig(name='Custom QuartzNet')
+
+    # Generate default asr model config
+    builder = configs.EncDecCTCModelConfigBuilder(name='quartznet_15x5')
+
+    # set model global values
+    builder.set_repeat(5)
```

Comment:
shouldn't this already be 5 if config is quartznet_15x5?

Reply:
if yes, let's remove this line

Reply:
Hmm good point. The default is 1 inside the config class since that is generic, but the builder should auto set this to 5 if its a valid config name.

Reply:
Done